src/main.cpp

```diff
@@ -1314,12 +1314,27 @@ bool ReadBlockFromDisk(CBlock& block, const CBlockIndex* pindex)
 
 CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams)
 {
-    int halvings = nHeight / consensusParams.nSubsidyHalvingInterval;
+    CAmount nSubsidy = 50 * COIN;
+
+    // Mining slow start
+    if (nHeight < consensusParams.nSubsidySlowStartInterval / 2) {
+        // Subsidy is ramped up linearly
+        nSubsidy /= consensusParams.nSubsidySlowStartInterval;
+        nSubsidy *= nHeight;
+        return nSubsidy;
+    } else if (nHeight < consensusParams.nSubsidySlowStartInterval) {
+        // Subsidy is ramped up linearly, skipping the 25 ZEC payout
```

Comment:
Note: the ``25`` here is sensitive to per-block subsidy, right? If we're changing block target time but striving to keep monetary base schedule roughly the same in realtime, we'll need to alter per-block subsidy.